### DISCUSSION TITLE
(Vite) Can I extract to CSS file

### DISCUSSION THREAD

**Discussion by jd-solanki** [vanilla-extract-css/vanilla-extract#220] [Q&A] [answered]
Hello,
I am using vite + vue and I want to extract the styles to the CSS file like `treat`. (as `treat` don't support vite)

Regards.

**Comment by mattcompiles** [comment, marked as answer]
The `@vanilla-extract/vite-plugin` should handle this for you. I haven't used Vue before so I'm not sure if there are any special requirements.
